.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - run: npm ci
+
+  publish-npm:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - run: npm publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```

Comment:
I think this should be removed?
It seems that secrets get removed on console output, yes, but if I write the secret in a file, like `echo ${{secrets.NPM_TOKEN}} > ./mySecret`, and after that I export it as an artifact, then the secret is completely visible to anyone... Isn't it?

Reply:
See https://github.com/aditosoftware/eslint-config-adito-platform/pull/1#discussion_r1754083657 and  [the example node.js publish pipeline](https://github.com/aditosoftware/eslint-config-adito-platform/new/main?filename=.github%2Fworkflows%2Fnpm-publish.yml&workflow_template=ci%2Fnpm-publish)

Reply:
So we have to have an eye on contributors, that change our workflows in pullrequests, right?

Reply:
Yes we do have. 
We need to check the commits and see their changed workflows and only if we think they are good, then we are running those.